backendRoot/src/main/java/com/virtuslab/gitmachete/backend/root/BranchLayoutSaverFactory.java

```diff
@@ -0,0 +1,14 @@
+package com.virtuslab.gitmachete.backend.root;
```

Comment:
I think this class should live in `branchLayoutImpl`, same for the parser factory